docs/multi-cluster-mesh/active-passive/multi-cluster-active-passive.md

```diff
@@ -0,0 +1,106 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
+## Objective:
```

Comment:
I'd only do a two cluster setup with different networks and (sometime later) another doc with two clusters for the same network setup. That's more comprehensible for the users I think, WDYT?

Reply:
done